src/main/scala/ch/openolitor/stammdaten/StammdatenCommandHandler.scala

```diff
@@ -792,14 +792,17 @@ trait StammdatenCommandHandler extends CommandHandler with StammdatenDBMappings
     DB readOnly { implicit session =>
       val personen = kunde.ansprechpersonen.map { person =>
         person.email match {
-          case Some(email) => stammdatenReadRepository.getPersonByEmail(email) match {
-            case Some(p) => if (p.kundeId != kundeId) {
-              Some(p)
-            } else {
-              None
-            }
-            case _ => None
-          }
+          case Some(email) =>
+            if (!email.isEmpty) {
```

Comment:
The project settings include a field specifying the possibility to have blank email addresses. This needs to be checked here, too!

Reply:
This is verified on the frontend. The backend is not enforcing to have a value on the "email" field.

Reply:
Hmm... ok.